src/paths.rs

```diff
@@ -1,16 +1,10 @@
-// Copyright 2021 Bart Ribbers
+// Copyright 2021-2022 Bart Ribbers & Leafish Developers
```

Comment:
Bart Ribbers is part of the Leafish developers, so i think "Leafish Developers" should be sufficient

Reply:
Sure. I will simplify copyrights, just that I was not sure if this would be fine.